media-gfx/icoutils/icoutils-0.31.1.recipe

```diff
@@ -0,0 +1,73 @@
+# FIXME: extresso and genresscript won't work because of missing libwww-perl
+# and that won't compile because reasons
+SUMMARY="Tools for extracting and converting Windows icon and cursor files"
+DESCRIPTION="A set of command-line programs for extracting and converting \
+images in Microsoft Windows icon and cursor files (usually with the extension \
+.ico or .cur, but they can also be embedded in executables and DLL libraries."
+HOMEPAGE="http://nongnu.org/icoutils/"
+COPYRIGHT="1998-2017 Frank Richter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://savannah.nongnu.org/download/icoutils/icoutils-$portVersion.tar.bz2"
+CHECKSUM_SHA256="26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+PATCHES="icoutils-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	icoutil$secondaryArchSuffix = $portVersion
```

Comment:
When I get these "No such file or directory:  ... /work-xxx" then I just remove the work-* directories.

Regarding the PATCH function, I do agree that it does not make any difference at the end.
But I tend to prefer a simple PATCH function.
BTW, if you had a patchset and the remove it, you need to remove the work-* directory.